src/main/java/frc/robot/commands/intake/RunIntakeCommand.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class RunIntakeCommand extends Command
+ {
+  /** Creates a new RunIntakeCommand. */
+
+  private IntakeSubsystem m_intakeSubsystem;
+  double m_Percent;
+
+  public RunIntakeCommand(IntakeSubsystem intake, double percent)
+  {
+    this.m_intakeSubsystem = intake;
+    this.m_Percent = percent;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(intake);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() 
+  {
+        m_intakeSubsystem.setIntakePercent(this.m_Percent);
+
```

Comment:
why extra indent and whitespace. We need to be reading through our code and looking for formatting errors before requesting reviews.